signup_utils.py

```diff
@@ -7,11 +7,19 @@ def process_sign_up():
     ver_pass_text = request.form['ver_pass']
     username_text = request.form['name']
 
-    if pass_text != ver_pass_text or not check_valid_username(username_text):
-        return False
-    db_management.create_new_user(username_text, pass_text)
-    return True
+    if pass_text != ver_pass_text:
```

Comment:
again, why use "text" in the name?

Reply:
changed the variable names to "password", "verification_password", and "username"